src/main/java/com/cisco/trex/stateful/api/lowlevel/ASTFProgram.java

```diff
@@ -404,8 +439,9 @@ public void setKeepAliveMsg(int msec) {
    * set var command
    *
    * @param varId
-   * @param value
+   * @param value @Deprecated use method with Long instead
    */
+  @Deprecated
   public void setVar(String varId, int value) {
     addVar(varId);
     fields.get(COMMANDS).add(new ASTFCmdSetVal(varId, (long) value));
```

Comment:
the constructor of ASTFCmdSetVal(String, Long) can be changed to ASTFCmdSetVal(String, long) to keep alignment with other classes